ee/tabby-webserver/docs/api_spec.md

```diff
@@ -141,3 +141,27 @@ The `Content-Type` for successful response is always `application/json`.
   ]
 }
 ```
+
+## OAuth api: `/oauth_callback`
+
+### GitHub
+
+**URL:** `/oauth_callback/github`
+
+**Method:** `GET`
+
+**Request example:**
+
+```shell
+curl --request GET \
+  --url http://localhost:8080/oauth_callback/github?code=1234567890
+```
+
+**Response example:**
```

Comment:
response should be an 302 temp redirect with accessToken / refreshToken query parameters.